NTupleMaker/src/Synch17GenTree.cc

```diff
@@ -57,7 +57,7 @@ void Synch17GenTree::ReadInit(TTree *tree)
    fCurrent = -1;
    fChain->SetMakeClass(1);
 
-   fChain->SetBranchAddress("Higgs_pt", &Higgs_pt, &b_Higgs_pt);
+   fChain->SetBranchAddress("GEN_Higgs_pt", &Higgs_pt, &b_Higgs_pt);
```

Comment:
This is the code regarding the gen level tree, I do not understand why you modified the name here and only for the Higgs pt.
Unless you had particular reasons, I would suggest to undo the modification.

Reply:
Hi,
it is a relic of an (unfinished) effort to smoothly store generator level observables in the RECO tree, for events where RECO is obeyed. But then they needed to have different name.
Since it doesn't hurt perhaps we can keep for now..
Best
Merijn

Reply:
It seems to work btw but I need to do some cross checks still

Reply:
Please let me know if this cross checks work fine

Reply:
OK I put it on my to do list.